{
  "path": "/documentation/foundation/nsexceptionname/1416220-internalinconsistencyexception",
  "type": "Type Property",
  "name": "internalInconsistencyException",
  "desc": "Name of an exception that occurs when an internal assertion fails and implies an unexpected condition within the called code.",
  "items": [],
  "declaration": "static let internalInconsistencyException: NSExceptionName"
}